### PR TITLE
Switch the rest of the PR jobs to prow.

### PR DESCRIPTION
**Pull request by spxtr**
Now that we can link to the correct gubernator location. Please do not merge for me.

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/test-infra/687)

### CODE REVIEW

**Comment by rmmh** [comment]
nit: rename ciongke/ directory

File: ciongke/cmd/test-pr/main.go [hunk 1]
```diff
@@ -172,18 +180,29 @@ func (c *testClient) TestPR() error {
...
 		}
+		result, err = c.JenkinsClient.Status(b)
 	}
 	return nil
 }
 
+func (c *testClient) guberURL(number int) string {
+	url := guberBase
+	if c.RepoOwner != "kubernetes" {
+		url = fmt.Sprintf("%s/%s/%s", url, c.RepoOwner, c.RepoName)
```
Comment (rmmh) [inline ciongke/cmd/test-pr/main.go]: not quite. "%s/%s_%s" is the expected format, here.

Comment (spxtr) [inline ciongke/cmd/test-pr/main.go]: Fixed.

**Comment by spxtr** [comment]
Will rename in a future PR.

**Comment by fejta** [comment]
Please add me back if you need my input

**Comment by spxtr** [comment]
OK, this is ready to go in.

**Comment by rmmh** [comment]
LGTM

**Pull request merged**